src/main.rs

```diff
@@ -65,7 +65,7 @@ enum Command {
 }
 
 #[derive(Parser, Debug)]
-#[command(author, version, about, long_about, rename_all = "kebab-case")]
+#[command(author, version=env!("VERSION"), about, long_about, rename_all = "kebab-case")]
```

Comment:
the version is set through an environment variable.
The build will fail if VERSION is not available at build time.